server/graphql/schema/index.js

```diff
@@ -33,13 +33,14 @@ module.exports = buildSchema(`
 
   type Bounty {
     createdDate: Object!
-    fundedAmount: Int!
+    fundedAmount: Float!
     id: ID!
     isApproved: Boolean!
     issueId: ID!
     name: String!
     pullRequestUrl: String!
     rep: String!
+    repoPayout: Boolean!
```

Comment:
Maybe we should name something that sounds like a boolean. Not clear what this is.

Reply:
I'll update to `repoPayoutExists`